src/XCCDF_POLICY/xccdf_policy.c

```diff
@@ -981,6 +979,52 @@ _xccdf_policy_rule_evaluate(struct xccdf_policy * policy, const struct xccdf_rul
 		return _xccdf_policy_report_rule_result(policy, result, rule, NULL, XCCDF_RESULT_NOT_SELECTED, NULL);
 	}
 
+	bool all_requires_met = true;
```

Comment:
I just reviewed the spec and it's not mentioned in the resolving workflow. Maybe we have to do it for every evaluated even if the benchmark has been resolved. IMO that's an issue in the spec, looks like the bolted on "requires" after they had the resolve workflow set-up.